skinny_orm/orm.py

```diff
@@ -1,10 +1,25 @@
 from skinny_orm.sqlite_orm import SqliteOrm
+import sqlite3
 
 
 class Orm:
+    def __new__(self, connection, create_tables_if_not_exists=True, parse_fields=True, connection_type="sqlite3"):
+        if isinstance(connection, str):
+            if connection_type=="sqlite3":
+                self.connection = sqlite3.connect(connection)
```

Comment:
Et ici vu que l'instance n'existe pas encore, "self.connection" est affecté à la class donc ce n'est pas bon aussi.
Je pense qu'il vaut mieux laisser l'injection de dépendance sans créer de connexion en cachette. "Explicit is better than implicit" ^^